### PR TITLE
Korean Locale

### PR DESCRIPTION
**Pull request by geralch**
**What kind of change does this PR introduce?**
Korean Locale

**What is the current behaviour?**
Doesn't have this Locale

**What is the new behaviour?**
Now it has Korean Locale

**Checklist**:
- [ ] Documentation N/A
- [ ] Tests N/A
- [x] Ready to be merged

### CODE REVIEW

File: src/locales/ko-KR.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "todayButton": "오늘",
+  "nextMonth": "다음 달",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:3]:
```suggestion
  "nextMonth": "다음달",
```

File: src/locales/ko-KR.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "previousMonth": "지난달",
+  "nextYear": "내년",
+  "previousYear": "전년도",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:6]:
```suggestion
  "previousYear": "작년",
```

File: src/locales/ko-KR.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "일요일",
+    "월요일",
+    "화요일",
+    "수요일",
+    "목요일",
+    "금요일",
+    "토요일"
+  ],
+  "months": [
+    "일월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:17]:
```suggestion
    "1월",
```

File: src/locales/ko-KR.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "이월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:18]:
```suggestion
    "2월",
```

File: src/locales/ko-KR.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "삼월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:19]:
```suggestion
    "3월",
```

File: src/locales/ko-KR.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    "사월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:20]:
```suggestion
    "4월",
```

File: src/locales/ko-KR.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "오월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:21]:
```suggestion
    "5월",
```

File: src/locales/ko-KR.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    "유월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:22]:
```suggestion
    "6월",
```

File: src/locales/ko-KR.json [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    "칠월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:23]:
```suggestion
    "7월",
```

File: src/locales/ko-KR.json [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+    "팔월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:24]:
```suggestion
    "8월",
```

File: src/locales/ko-KR.json [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+    "구월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:25]:
```suggestion
    "9월",
```

File: src/locales/ko-KR.json [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+    "시월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:26]:
```suggestion
    "10월",
```

File: src/locales/ko-KR.json [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+    "십일월",
```
Comment (luffy-song) [inline src/locales/ko-KR.json:27]:
```suggestion
    "11월",
```

File: src/locales/ko-KR.json [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+    "십이월"
```
Comment (luffy-song) [inline src/locales/ko-KR.json:28]:
```suggestion
    "11월"
```

**Comment by arthurdenner** [comment]
Closed due to inactivity.

**Pull request closed without merging**